plugins/woocommerce/src/Blocks/BlockPatterns.php

```diff
@@ -141,9 +141,12 @@ public function register_ptk_patterns() {
 
 		$patterns = $this->ptk_patterns_store->get_patterns();
 		if ( empty( $patterns ) ) {
-			wc_get_logger()->warning(
-				__( 'Empty patterns received from the PTK Pattern Store', 'woocommerce' ),
-			);
+			// Check if there's a scheduled 'fetch_patterns' action.
```

Comment:
The name of the function is quite self-explanatory. I suggest either adding more context to clarify its purpose or removing the comment altogether.